readthedocs/projects/models.py

```diff
@@ -1859,6 +1860,12 @@ def add_features(sender, **kwargs):
                 'Do not overwrite context vars in conf.py with Read the Docs context',
             ),
         ),
+        (
+            SKIP_SPHINX_HTML_THEME_PATH,
+            _(
+                "On Sphinx < 6, do not define html_theme_path",
```

Comment:
Less passive voice here makes items a little easier to scan. The features list is a bit unwieldy when you need to interact with it. Also, typo here I assume.

```suggestion
                "Do not define html_theme_path on Sphinx >= 6.0",
```

Reply:
In [the other comment](https://github.com/readthedocs/readthedocs.org/pull/9654#discussion_r1085629807), you indicate that a user-defined configuration value is better than a feature flag?

Reply:
Sorry -- to clarify: this fix falls solidly into feature flag usage. That means that at some point core team will expand our testing sample in include more projects, in order to get more feedback. This won't be an option that we expect users can ask us to enable for their project.